src/Microsoft.AspNet.Identity/SignInResult.cs

```diff
@@ -40,74 +32,62 @@ public class SignInResult
         ///     Static success result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult Success
-        {
-            get { return _success; }
-        }
+        public static SignInResult Success => new SignInResult { Succeeded = true };
 
         /// <summary>
         ///     Static failure result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult Failed
-        {
-            get { return _failed; }
-        }
+        public static SignInResult Failed => new SignInResult();
```

Comment:
Good point. My fault. I suggested @suhasj to look into something like this just to make this look simpler.